en/docs/get-started/how-to-guides/ai-code-generation.md

```diff
@@ -0,0 +1,185 @@
+# AI Integration Generation
+
+Did you know that the WSO2 Micro Integrator component of WSO2 Integrator includes an AI-powered assistant to help you effortlessly generate integration flows?
+In this guide, we are generating an API to retrieve weather information for a specified city and send it via email to a designated recipient.
+
+## Let's get started!
+
+### Prerequisites
+
+You need Visual Studio Code (VS Code) with the <a target="_blank" href="https://marketplace.visualstudio.com/items?itemName=WSO2.micro-integrator">Micro Integrator for VS Code</a> extension installed.
+
+!!! Info
+    See the [Install Micro Integrator for VS Code]({{base_path}}/develop/mi-for-vscode/install-wso2-mi-for-vscode) documentation to learn how to install Micro Integrator for VS Code.
+
+### Step 1: Create an integration project
+
+{!includes/create-new-project.md!}
+
+4. In the **Project Creation Form**, enter `AICodeDemo` as the **Project Name**.
```

Comment:
Why this is numbered as 4?

Reply:
First 3 steps are added by the include `create-new-project.md`.

Reply:
Ack.